src/PrestaShopBundle/Event/Dispatcher/NullDispatcher.php

```diff
@@ -41,14 +41,30 @@ public function addSubscriber(EventSubscriberInterface $subscriber)
     {
     }
 
+    /**
+     * @param string $eventName
+     * @param \Symfony\Component\EventDispatcher\Event|null $event
+     *
+     * @return \Symfony\Component\EventDispatcher\Event|void
+     */
     public function dispatch($eventName, \Symfony\Component\EventDispatcher\Event $event = null)
```

Comment:
Same here, why not importing with `use`?

Reply:
Fixed